## USB port
port: /dev/ttyUSB0


##### Calibration ####
### accelerometer
accel_x_min: -250.0
accel_x_max: 250.0
accel_y_min: -250.0
accel_y_max: 250.0
accel_z_min: -250.0
accel_z_max: 250.0

### magnetometer
# standard calibration
magn_x_min: -600.0
magn_x_max: 600.0
magn_y_min: -600.0
magn_y_max: 600.0
magn_z_min: -600.0
magn_z_max: 600.0

# extended calibration
calibration_magn_use_extended: false
magn_ellipsoid_center: [0, 0, 0]
magn_ellipsoid_transform: [[0, 0, 0], [0, 0, 0], [0, 0, 0]]

# AHRS to robot calibration
imu_yaw_calibration: 0.0

### gyroscope
gyro_average_offset_x: 0.0
gyro_average_offset_y: 0.0
gyro_average_offset_z: 0.0

# frame_id for IMU message
frame_id: base_imu_link

# Remove acceleration biases
remove_x_accel_bias: False
remove_y_accel_bias: True
remove_z_accel_bias: True

# threshold under which acceleration values go to 0
thresh: 0.1
